config/routes.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+Rails.application.routes.draw do
+  devise_for :users
+  root 'episodes#index'
+
+  resources :episodes
```

Comment:
Do we need all routes for `resources :episodes`? Edit, destroy? It's admin permission. So mb we need to move it to admin scope

Reply:
We authorize user before every action.